kotlinx-coroutines-core/common/src/selects/Select.kt

```diff
@@ -177,16 +177,6 @@ public interface SelectInstance<in R> {
  * corresponding non-suspending version that can be used with a regular `when` expression to select one
  * of the alternatives or to perform the default (`else`) action if none of them can be immediately selected.
  *
- * | **Receiver**     | **Suspending function**                       | **Select clause**                                | **Non-suspending version**
- * | ---------------- | --------------------------------------------- | ------------------------------------------------ | --------------------------
- * | [Job]            | [join][Job.join]                              | [onJoin][Job.onJoin]                             | [isCompleted][Job.isCompleted]
- * | [Deferred]       | [await][Deferred.await]                       | [onAwait][Deferred.onAwait]                      | [isCompleted][Job.isCompleted]
- * | [SendChannel]    | [send][SendChannel.send]                      | [onSend][SendChannel.onSend]                     | [offer][SendChannel.offer]
- * | [ReceiveChannel] | [receive][ReceiveChannel.receive]             | [onReceive][ReceiveChannel.onReceive]            | [poll][ReceiveChannel.poll]
- * | [ReceiveChannel] | [receiveOrNull][ReceiveChannel.receiveOrNull] | [onReceiveOrNull][ReceiveChannel.onReceiveOrNull]| [poll][ReceiveChannel.poll]
- * | [Mutex]          | [lock][Mutex.lock]                            | [onLock][Mutex.onLock]                           | [tryLock][Mutex.tryLock]
- * | none             | [delay]                                       | [onTimeout][SelectBuilder.onTimeout]             | none
- *
  * This suspending function is cancellable. If the [Job] of the current coroutine is cancelled or completed while this
```

Comment:
Without a list of select clauses in this doc, a causal documentation reader who stumbles up the the `select` function would have zero clues on what are those "select clauses" that this doc talks about and how `select` exception can be actually used. It is non-discoverable without a list of various supported select clauses.

Reply:
I've returned the list back but removed "non-suspending functions" column. It's not really informative, but with the current names (`[onReceiveCatching][ReceiveChannel.onReceiveCatching]`) this table became as wide as 200 symbols, rendering it unreadable from IDE.

This documentation still needs some love, e.g. the table in its current state is not properly rendered in IDEA quick-KDoc, but it seems to be out of the scope